types/express-redis-cache/express-redis-cache-tests.ts

```diff
@@ -0,0 +1,105 @@
+import * as express from 'express';
+import * as redis from 'redis';
+import * as expressRedisCache from 'express-redis-cache';
```

Comment:
since express-redis-cache uses the commonjs-specific `export =` syntax, it should be imported with commonjs-specific syntax: `import expressRedisCache = require('express-redis-cache')`

Reply:
gotcha, changes made